types/gravatar-url/index.d.ts

```diff
@@ -3,14 +3,10 @@
 // Definitions by: Ivan Gabriele <https://github.com/ivangabriele>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export = GravatarUrl;
-
-declare function GravatarUrl(email: string, options?: GravatarUrl.Options): string;
-
-declare namespace GravatarUrl {
-    interface Options {
-        default?: string;
-        rating?: 'g' | 'pg' | 'r' | 'x';
-        size?: number;
-    }
+export interface Options {
+    default?: string;
+    rating?: 'g' | 'pg' | 'r' | 'x';
+    size?: number;
 }
+
+export default function GravatarUrl(email: string, options?: Options): string;
```

Comment:
Please see https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
Here is what the package looks like at runtime:

```sh
>npm install gravatar-url
+ gravatar-url@2.0.0
added 3 packages and removed 4 packages in 3.155s

>node
> var g = require("gravatar-url");
undefined
> g;
[Function]
```

Reply:
@mhegazy Thank you ! I didn't know this option ;)